.github/workflows/tests.yml

```diff
@@ -5,8 +5,8 @@ name: CI
 on:
```

Comment:
Can delete this file, we are not using github actions here

Reply:
👌